stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messageoptions/MessageOptions.kt

```diff
@@ -217,15 +215,6 @@ public fun defaultMessageOptionsState(
                 titleColor = ChatTheme.colors.errorAccent
             )
         } else null,
-        if (!isOwnMessage) {
-            MessageOptionItemState(
```

Comment:
More difficult to properly deprecate this option in Compose so I opted in for removing it. Users have an easier time of rebuilding this themselves.